src/esn.inbox.libs/app/services/mailboxes/mailboxes-service.js

```diff
@@ -219,7 +226,13 @@ angular.module('esn.inbox.libs')
 
     function updateSharedMailboxCache() {
       return withJmapClient(function(jmapClient) {
-        return jmapClient.getMailboxes()
+        return jmapClient.mailbox_get({
+          accountId: Object.keys(jmapClient.getSession().accounts)[0],
```

Comment:
Ok, about this, I am wondering if the lib should support single account in session by default: if `accountId` is set to null in the parameter, it will automatically try to look for accountId value in stored session. The `getSession` would then not be used for now. If it is used though, it should return an immutable session.

Reply:
Added in jmap-client-ts: https://github.com/OpenPaaS-Suite/jmap-client-ts/pull/2